hasher-matcher-actioner/hmalib/lambdas/actions/action_evaluator.py

```diff
@@ -53,7 +54,7 @@ def get(cls):
         HMAConfig.initialize(os.environ["CONFIG_TABLE_NAME"])
         return cls(
             actions_queue_url=os.environ["ACTIONS_QUEUE_URL"],
-            reactions_queue_url=os.environ["REACTIONS_QUEUE_URL"],
+            writebacks_queue_url=os.environ["WRITEBACKS_QUEUE_URL"],
```

Comment:
Not seeing the compensating change in terraform code for the change in this environment variable.

Reply:
Added and finished full e2e test